sympy/series/limits.py

```diff
@@ -360,4 +414,6 @@ def set_signs(expr):
             if r is None:
                 return self
 
+        if z0.is_Symbol and not z0 in e.free_symbols:
+            return _limit_for_symbolic_point(r, z0, cdir)
```

Comment:
Can this just be called once sometime before gruntz, instead of having 3 different locations to call it?

Reply:
Yeah but only if somehow I could implement the suggestion below , otherwise would have to use it seperately for both the `is_meromorphic` and ` z0 is S.Infinity` blocks